h2o-py/tests/testdir_algos/gam/pyunit_PUBDEV_8163_gam_cv_fold_column.py

```diff
@@ -60,6 +60,7 @@ def test_gam_cv_fold_columns():
     h2o_model_fold_column = H2OGeneralizedAdditiveEstimator(family='multinomial',
                                                             gam_columns=["C6", "C7", "C8"],
                                                             scale=[1, 1, 1],
+                                                            seed=1234,
                                                             num_knots=numKnots,
                                                             knot_ids=[frameKnots1.key, frameKnots2.key,
                                                                       frameKnots3.key])
```

Comment:
In my understanding, this test was used to compare if setting the `nfolds` work the same way as using precomputed folds using `.kfold_column`. The issue here was that it used to compare `MODULO` fold assignment with `RANDOM` fold assignment.
I think the convergence to the same coefficients was just luck.